dissect/target/plugins/os/unix/linux/fortios/_os.py

```diff
@@ -277,3 +421,134 @@ def _parse_config(fh: TextIO) -> Iterator[list[str]]:
         if parts and not string:
             yield parts
             parts = []
+
+
+def parse_version(input: str) -> str:
+    """Attempt to parse the config FortiOS version to a readable format
+
+    The input ``FGVM64-7.4.1-FW-build2463-230830:opmode=0:vdom=0`` would
+    return the following output: ``FortiGate VM 7.4.1 (build 2463, 2023-08-30)``.
+
+    Resources:
+        - https://support.fortinet.com/Download/VMImages.aspx
+    """
+
+    PREFIXES = {
+        "FGV": "FortiGate VM",  # FGVM64
+        "FGT": "FortiGate",  # can also be FGT-VM in 4.x/5.x
+        "FMG": "FortiManager",
+        "FAZ": "FortiAnalyzer",
+        "FFW": "FortiFirewall",
+        "FOS": "FortiOS",
+        "FWB": "FortiWeb",
+        "FAD": "FortiADC",
+    }
+
+    try:
+        version_str = input.split(":", 1)[0]
+        type, version, _, build_num, build_date = version_str.rsplit("-", 4)
+
+        build_num = build_num.replace("build", "build ", 1)
+        build_date = datetime.strptime(build_date, "%y%m%d").strftime("%Y-%m-%d")
+        type = PREFIXES.get(type[:3], type)
+
+        return f"{type} {version} ({build_num}, {build_date})"
+    except ValueError:
+        return input
+
+
+def local_groups_to_users(config_groups: dict) -> dict:
+    """Map FortiOS groups to a dict with usernames as key."""
+    user_groups = {}
+    for group, items in config_groups.items():
+        for user in items.get("member", []):
+            if user in user_groups:
+                user_groups[user].append(group)
+            else:
+                user_groups[user] = [group]
+    return user_groups
+
+
+def decrypt_password(b64_ciphertext: str) -> str:
```

Comment:
```suggestion
def decrypt_password(ciphertext: str) -> str:
```
Don't think the `b64_` prefix adds much. This looks a bit cleaner.